spec/rubocop/cop/rspec/example_wording_spec.rb

```diff
@@ -351,5 +351,57 @@
         end
       RUBY
     end
+
+    it 'corrects escaped single-quote' do
+      expect_offense(<<~'RUBY')
+        it 'should return foo\'s bar' do
+            ^^^^^^^^^^^^^^^^^^^^^^^^ Do not use should when describing your tests.
+        end
+      RUBY
+
+      expect_correction(<<~RUBY)
+        it "returns foo's bar" do
+        end
+      RUBY
+    end
+
+    it 'corrects escaped double-quote' do
+      expect_offense(<<~'RUBY')
+        it "should return \"foo\"" do
+            ^^^^^^^^^^^^^^^^^^^^^ Do not use should when describing your tests.
+        end
+      RUBY
+
+      expect_correction(<<~'RUBY')
+        it "returns \"foo\"" do
+        end
+      RUBY
+    end
+
+    it 'corrects %(...) quote' do
+      expect_offense(<<~RUBY)
+        it %q(should return foo (bar)) do
```

Comment:
What if there’s a single double quote inside?

Reply:
In that case, the `should ` part is simply removed. Since the `node.loc.end` (`"` ) is not included, `String#inspect` is not used, and the behavior is the same as before.

I added a test case for that 👌

Reply:
Good point. Whether or not `"` is included, it is autocorrected with `String#inspect` if `%q(...)` includes `)`.
I added more test cases for that 👌 

```
when message includes `)` in `%q(...)` string literal
  corrects message with `String#inspect`
when message includes `"` in `%q(...)` string literal
  corrects message with direct substring replacement
when message includes `"` and `)` in `%q(...)` string literal
  corrects message with `String#inspect`
```